jcommune-view/jcommune-web-controller/src/test/java/org/jtalks/jcommune/web/controller/TopicControllerTest.java

```diff
@@ -205,7 +205,7 @@ public void createTopicShouldNotPassAndMustShowTopicErrorIfItIsInvalid() throws
         ModelAndView mav = controller.createTopic(getDto(), result, BRANCH_ID);
 
         verify(branchService).get(BRANCH_ID);
-        verify(breadcrumbBuilder).getForumBreadcrumb(branch);
+        verify(breadcrumbBuilder).getNewTopicBreadcrumb(branch);
```

Comment:
Hm.. actually this test should check attributes instead of ModelAndView instead of breadcrumb builder..
@mixas27 what other controllers do you think should be covered?